ttnn/cpp/ttnn/tensor/types.cpp

```diff
@@ -417,6 +417,10 @@ uint64_t SimpleShape::volume() const {
 
 namespace ttnn::types {
 
-uint32_t Shape::operator[](std::int64_t index) const { return this->value.without_padding()[index]; }
+uint32_t Shape::operator[](std::int64_t index) const {
+    const auto dimension = value[index];
+    auto [front_pad, back_pad] = value.padding().pad_dimensions_[index];
```

Comment:
do you have to handle negative index here?

Reply:
Fixed